modules/update-upgrading-cli.adoc

```diff
@@ -27,52 +26,57 @@ you want to apply:
+Channel: stable-scos-4
+
+Updates to 4.20:
+VERSION               ISSUES
+4.20.0-okd-scos.ec.14 no known issues relevant to this cluster
+
+Updates to 4.19:
+VERSION            ISSUES
+4.19.0-okd-scos.17 no known issues relevant to this cluster
 ----
 endif::openshift-origin[]
-ifdef::openshift-origin[]
 .Example output
 [source,terminal]
 ----
-Cluster version is 4.13.0-0.okd-2023-10-28-065448
+Upstream is unset, so the cluster will use an appropriate default.
+Channel: stable-4.13 (available channels: candidate-4.12, candidate-4.13, eus-4.12, eus-4.14, fast-4.12, fast-4.13, stable-4.12, stable-4.13)
 
-Upstream: https://amd64.origin.releases.ci.openshift.org/graph
-Channel: stable-4
+Updates to 4.13:
+
+Version: 4.13.50
+Image: quay.io/openshift-release-dev/ocp-release@sha256:6afb11e1cac46fd26476ca134072937115256b9c6360f7a1cd1812992c065f02
+Reason: AdminAckRequired
+Message: Kubernetes 1.26 and therefore OpenShift 4.13 remove several APIs which require admin consideration. Please see the knowledge article https://access.redhat.com/articles/6958394 for details and instructions.
 
-Recommended updates:
+Version: 4.13.49
+Image: quay.io/openshift-release-dev/ocp-release@sha256:ab6f574665395d809511db9dc57764358278538eaae248c6d199208b3c30ab7d
+Reason: AdminAckRequired
+Message: Kubernetes 1.26 and therefore OpenShift 4.13 remove several APIs which require admin consideration. Please see the knowledge article https://access.redhat.com/articles/6958394 for details and instructions.
 
-  VERSION                        IMAGE
-  4.14.0-0.okd-2024-01-06-084517 registry.ci.openshift.org/origin/release@sha256:c4a6b6850701202f629c0e451de784b02f0de079650a1b9ccbf610448ebc9227
-  4.14.0-0.okd-2023-11-14-101924 registry.ci.openshift.org/origin/release@sha256:72d40c51e7c4d1b9c31e9b0d276d045f1b2b93def5ecee49186df856d40bcb5c
-  4.14.0-0.okd-2023-11-12-042703 registry.ci.openshift.org/origin/release@sha256:2242d1df4e4cbcc0cd27191ab9ad5f55ac4f0c60c3cda2a186181a2435e3bd00
-  4.14.0-0.okd-2023-10-28-073550 registry.ci.openshift.org/origin/release@sha256:7a6200e347a1b857e47f2ab0735eb1303af7d796a847d79ef9706f217cd12f5c
+Updates to 4.12:
+VERSION     ISSUES
+4.12.64     no known issues
+4.12.63     no known issues
+And 43 older 4.12 updates you can see with '--show-outdated-releases' or '--version VERSION'.
 ----
-endif::openshift-origin[]
-+
+
 [NOTE]
 ====
-* If there are no recommended updates, updates that have known issues might still be available.
-See _Updating along a conditional update path_ for more information.
-ifndef::openshift-origin[]
-* For details and information on how to perform a `Control Plane Only` update, please refer to the _Preparing to perform a Control Plane Only update_ page, listed in the Additional resources section.
-endif::openshift-origin[]
+To determine whether a specific version is recommended for your update, run the `oc adm upgrade recommend --version` command. For example, `oc adm upgrade recommend --version 4.12.51`.
```

Comment:
This line (with an OKD version like `--version 4.19.0-okd-scos.17`) is probably worth including in the OKD/origin branch of these docs as well.